packages/suite/src/support/messages.ts

```diff
@@ -1357,6 +1357,42 @@ const definedMessages = defineMessages({
             'If you forget your passphrase, your wallet is lost for good. There is no way to recover your funds.',
         id: 'TR_DEVICE_SETTINGS_PASSPHRASE_DESC_MORE',
     },
+    TR_DEVICE_SETTINGS_SAFETY_CHECKS_TITLE: {
+        defaultMessage: 'Safety checks',
+        id: 'TR_DEVICE_SETTINGS_SAFETY_CHECKS_TITLE',
+    },
+    TR_DEVICE_SETTINGS_SAFETY_CHECKS_DESC: {
+        defaultMessage: 'Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.',
+        id: 'TR_DEVICE_SETTINGS_SAFETY_CHECKS_DESC',
+    },
+    TR_DEVICE_SETTINGS_SAFETY_CHECKS_BUTTON: {
+        defaultMessage: 'Edit',
+        id: 'TR_DEVICE_SETTINGS_SAFETY_CHECKS_BUTTON',
+    },
+    TR_SAFETY_CHECKS_MODAL_TITLE: {
+        defaultMessage: 'Safety checks',
+        id: 'TR_SAFETY_CHECKS_MODAL_TITLE',
+    },
+    TR_SAFETY_CHECKS_STRICT_LEVEL: {
+        defaultMessage: 'Strict',
+        id: 'TR_SAFETY_CHECKS_STRICT_LEVEL',
+    },
+    TR_SAFETY_CHECKS_STRICT_LEVEL_DESC: {
+        defaultMessage: 'Strict level description',
+        id: 'TR_SAFETY_CHECKS_STRICT_LEVEL_DESC',
+    },
+    TR_SAFETY_CHECKS_PROMPT_LEVEL: {
+        defaultMessage: 'Prompt',
+        id: 'TR_SAFETY_CHECKS_PROMPT_LEVEL',
+    },
+    TR_SAFETY_CHECKS_PROMPT_LEVEL_WARNING: {
+        defaultMessage: 'We really don’t recommend this.',
+        id: 'TR_SAFETY_CHECKS_PROMPT_LEVEL_WARNING',
+    },
+    TR_SAFETY_CHECKS_PROMPT_LEVEL_DESC: {
+        defaultMessage: 'Prompt level description',
+        id: 'TR_SAFETY_CHECKS_PROMPT_LEVEL_DESC',
+    },
```

Comment:
Shouldn't the translations have better default text that is closer to what the actual values will be?

Or what's the best way to put there? I can imagine that an empty string as something more defnsive might be better in some cases.

Reply:
Answered on Slack.

> Shouldn't the translations have better default text that is closer to what the actual values will be?

They should. I'm already putting together good defaults for the keys in this PR with other colleagues.

> Or what's the best way to put there? I can imagine that an empty string as something more defnsive might be better in some cases.

For the time being the attitude is that the dev writes a text that is good enough and doesn't bother with polishing the copy. That is left for someone who can do it properly later via Crowdin.

Reply:
I think it there has to be some meaningful content but also might be changed later from Crowding and be synced back to the app. I personally wouldn't use an empty string.

Reply:
A good enough translations added in https://github.com/trezor/trezor-suite/pull/3590/commits/91681f244202c2730d433b7aef8d89466d999c4c.

Resolving.